src/validator/has-length-greater-than-or-equal-to.ts

```diff
@@ -0,0 +1,37 @@
+import {STRule} from '../rule/rule';
+import {STRuleFn} from '../rule/fn';
+import {STRuleModifiers} from '../rule/modifiers';
+import {STRuleNode} from '../rule/node';
+import {STRuleNodeType} from '../rule/node-type';
+import {greaterThanOrEqualToFn} from './is-greater-than-or-equal-to';
+
+export type STOpHasLengthGreaterThanOrEqualTo<CallerType> = (a: number) => CallerType;
+
+export const hasLengthGreaterThanOrEqualTo = (curr: any, target: number): boolean => {
+	if (curr.length <= -1) {
+		return false;
+	}
+
+	if (!curr.length) {
```

Comment:
See above detailed comments elsewhere for more info, but update this to be a number type check.